### PR TITLE
Development

### PR DESCRIPTION
**Pull request by shjang7**
last change

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -19,61 +15,117 @@
     <link rel="stylesheet" href="css/design.css" type="text/css" />
 
     <script src="js/javascript.js" type="text/javascript"></script>
```
Comment (zlayabekrija) [inline index.html]: Your js is empty, please remove it.

File: index.html [hunk 2]
```diff
@@ -19,61 +15,117 @@
...
-    <table>
-      <td>
-        <!-- left side -->
-        <tr>
+    <nav class="nav_bar"></nav>
+
+    <div class="container">
+      <header>
+        <div class="logo_icon">
+          <img class="logo" src="assets/logo.png" alt="YouTube" />
```
Comment (zlayabekrija) [inline index.html]: Should be moved more towards to middle.

File: index.html [hunk 3]
```diff
@@ -19,61 +15,117 @@
...
+        </div>
+
+        <div class="search_input">
+          <form action="">
+            <input type="text" placeholder="Search ..." size="80" />
+            <button><i class="fas fa-search"></i></button>
+          </form>
+        </div>
+
+        <div class="icons">
```
Comment (zlayabekrija) [inline index.html]: Icons should be bigger, spaced out, more towards the middle you can use class and apply class on them.

File: index.html [hunk 4]
```diff
@@ -19,61 +15,117 @@
...
+        </div>
+      </header>
+
+      <main>
+        <div class="curr_video">
           <iframe
-            width="560"
-            height="315"
-            src="https://www.youtube.com/embed/V74l_zS1x8E"
+            width="100%"
```
Comment (zlayabekrija) [inline index.html]: iframe should use only digits (no em,rem,px,% vw,vh etc)

File: index.html [hunk 5]
```diff
@@ -19,61 +15,117 @@
...
+          <p>32,312 Views</p>
+        </div>
+
+        <hr />
+
+        <div id="author">
+          <img
+            class="header"
+            src="http://www.mpp.govt.nz/themes/mpp/images/profile_placeholder.jpg"
+            alt=""
```
Comment (zlayabekrija) [inline index.html]: alt property shouldn't be empty.

File: index.html [hunk 6]
```diff
@@ -19,61 +15,117 @@
...
+        </div>
+
+        <div>
+          <h4>Comments</h4>
+
+          <form action="">
+            <input
+              type="text"
+              placeholder="Add a public comment.."
+              style="width:100%; align-items: center"
```
Comment (zlayabekrija) [inline index.html]: Please don't use inline styling.

File: index.html [hunk 7]
```diff
@@ -19,61 +15,117 @@
...
+                src="http://www.mpp.govt.nz/themes/mpp/images/profile_placeholder.jpg"
+              />
+              It is awsome!!<i class="fas fa-thumbs-up"></i
+              ><i class="fas fa-thumbs-down"></i>
+            </li>
+            <li>
+              <img
+                class="header"
+                src="http://www.mpp.govt.nz/themes/mpp/images/profile_placeholder.jpg"
+              />Incredible!!<i class="fas fa-thumbs-up"></i
```
Comment (zlayabekrija) [inline index.html]: same here, please space out the thumbs up and down icons.

File: index.html [hunk 8]
```diff
@@ -19,61 +15,117 @@
...
+      </main>
+
+      <aside>
+        <!--first sidebar vedio-->
+        <p>
+          Up next
+        </p>
+
+        <div>
+          <img src="./assets/sidbar.png" alt="" />
```
Comment (zlayabekrija) [inline index.html]: please fill up appropriate thumbnails instead of a screenshot.

File: css/design.css [hunk 9]
```diff
@@ -1,24 +1,118 @@
...
+  background-color: white;
+  overflow: hidden;
+  z-index: 5;
   height: 30px;
+
+  grid-area: header;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: 1rem;
```
Comment (zlayabekrija) [inline css/design.css]: it's better practice to give width property instead of padding left and right together.

File: css/design.css [hunk 10]
```diff
@@ -1,24 +1,118 @@
...
+  font-size: 0.8rem;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+li {
+  list-style: none;
+}
+nav {
+  grid-area: nav;
+  /* margin-left: 0.5rem; */
```
Comment (zlayabekrija) [inline css/design.css]: please delete the commented out properties from css

File: css/design.css [hunk 11]
```diff
@@ -1,24 +1,118 @@
...
+  grid-area: content;
+}
+
+aside {
+  grid-area: side;
+  margin-left: 1 rem;
+  margin-right: 0.5 rem;
+}
+
+/* aside img {
```
Comment (zlayabekrija) [inline css/design.css]: same here

File: css/design.css [hunk 12]
```diff
@@ -1,24 +1,118 @@
...
+  width: 150px;
+} */
+aside p {
+  font-size: 0.5 rem;
+  font-family: "Franklin Gothic Medium", "Arial Narrow", Arial, sans-serif;
+}
+
+.curr_video {
+  /* padding: 30px 10px 10px 20px; */
```
Comment (zlayabekrija) [inline css/design.css]: and here :)

File: css/design.css [hunk 13]
```diff
@@ -1,24 +1,118 @@
...
+  width: 90%;
+}
+
+/* .side_bar {
```
Comment (zlayabekrija) [inline css/design.css]: here as well :)

**Comment by kelvin8773** [comment]
We just fixed the issue  mention above, please help review again!

**Pull request merged**